apps/laboratory/src/pages/_document.tsx

```diff
@@ -5,7 +5,7 @@ export default function Document() {
   return (
     <Html lang="en">
       <Head />
-      <body>
+      <body style={{ minHeight: '100vh' }}>
```

Comment:
We need to apply this `min-heiught: 100vh` in `w3m-modal/index` file when calling `onScrollLock` function so it works correctly on all sites.